### DISCUSSION TITLE
Request for making the energy demand of district heating networks visible as KPI

### DISCUSSION THREAD

**Discussion by RonaldBerk** [Project-OMOTES/user_feedback#57] [Q&A]
Can we calculate how much power is consumed by the geothermal or other sources and make the result visible as KPI (in MWh)?

**Comment by ewoudwerkman** [comment]
@KobusVanRooyen this is something that needs to be added in the optimizer to calculate this KPI

**Comment by jarsarasty** [comment]
As discussed in the partner support meeting last week, the Seasonal COefficient of Performance (SCOP) could also be calculated. For geothermal sources, energy used for pumping should be included.

**Comment by jarsarasty** [comment]
Energy consumption and SCOP are complementary KPI's

**Comment by jarsarasty** [comment]
Hi @RonaldBerk ,
Based on our discussion, two feature requests have been created:

- [electricity consumption as KPI](https://github.com/Project-OMOTES/user_feedback/issues/67)
- [SCOP as KPI](https://github.com/Project-OMOTES/user_feedback/issues/68)
